oidc-api/src/main/java/uk/gov/di/authentication/oidc/helpers/RequestObjectToAuthRequestHelper.java

```diff
@@ -48,6 +49,9 @@ public static AuthenticationRequest transform(AuthenticationRequest authRequest)
             if (Objects.nonNull(jwtClaimsSet.getClaim("vtr"))) {
                 transformVtr(builder, jwtClaimsSet);
             }
+            if (Objects.nonNull(jwtClaimsSet.getClaim("claims"))) {
+                builder.customParameter("claims", jwtClaimsSet.getClaim("claims").toString());
+            }
```

Comment:
I don't think we want to add it as a custom parameter - I think we want to add it as the "claims" parameter, similarly how we do with state and nonce above